### ISSUE TITLE
Do you mind sharing insights on how you handle orders internally

### ISSUE THREAD

**Issue by derberg** [gatsbyjs/store.gatsbyjs.org#425]
Hey, folks. I love the idea you have here and how you handle this store. I was thinking about something like this for our AsyncAPI project that is part of Linux Foundation and we would like to explore how could we make it easier for our community to get access to swags.

Core question is how does it work in the background. I know you handle shop through Shopify, but who at the end handles order fulfillment, who actually gets and order and sends swags. Is there some other 3rd party service that you are using?

Thanks in advance for any insights 🙏🏼

**Comment by hkaur008** [comment]
yes there are i m sending swags to openforce mentees so i know there are services who will manufacture and deliver swags for you

**Comment by hkaur008** [comment]
Order in bulk , for getting discounts and ask for more discounts personally by durectly calling or having a meet with seller

**Comment by derberg** [comment]
@hkaur008 but is it a service that has API and it is automated, or do you have to handle it on your own. Like take the delivery address from a person and add it to the system? I want to avoid manual job

**Comment by hkaur008** [comment]
Okayy,  got it ✅

**Comment by hkaur008** [comment]
Can you just let me know what exactly you are looking for?

A swag vendor

A swag shipper

Or anything else

You want to give people swag and dont want to put manual effort


Hire  One single company to manufacture/package/ship swags

Send them name and address and other details
They will do it.

**Comment by derberg** [comment]
> Hire  One single company to manufacture/package/ship swags

well yes, and this is why I'm asking Gatsby folks to figure what service are they using for it 😄
